Documentation/devel/stage1-implementors-guide.md

```diff
@@ -135,3 +135,26 @@ Examples
     ]
 }
 ```
+
+##Filesystem Layout Assumptions
+
+The following pathes are reserved for the stage1 image, and they will be created during stage0.
+Users **SHOULD NOT** create or use these pathes when making a stage1 image.
+
+### opt/stage2
+
+This directory path is used for extracting the app's ACI.
+`rkt gc` also expects the app's rootfs to be mounted under this directory when
+overlayfs is used (e.g. `/var/lib/rkt/pods/$uuid/stage1/rootfs/opt/stage2/$appname/rootfs`).
```

Comment:
True at the moment, but `rkt gc` will soon not expect this anymore with #1856.

/cc @steveeJ